venues/auai.org/UAI/2019/Conference/python/get-pdfs.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import os
+import openreview
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print ('connecting to {0}'.format(client.baseurl))
+
+submission_invitation = 'auai.org/UAI/2019/Conference/-/Blind_Submission'
+submissions = openreview.tools.iterget_notes(client, invitation=submission_invitation)
+
+if not os.path.exists('../pdfs'):
```

Comment:
doesn't really matter if we're running this script ourselves, but if you're giving it to someone, I think it should try to create a pdfs directory wherever the user is running the script, not one level up. I also think it should be named something more unique, like "uai18-pdfs" or something (just in case they already have a pdfs directory wherever they're running the script).

(but again, doesn't really matter if we're running it ourselves)